lightning/src/ln/channel.rs

```diff
@@ -3238,6 +4212,92 @@ impl<SP: Deref> Channel<SP> where
 		Ok(())
 	}
 
+	#[cfg(dual_funding)]
+	pub fn commitment_signed_initial_v2<L: Deref>(&mut self, msg: &msgs::CommitmentSigned,
+		best_block: BestBlock, signer_provider: &SP, logger: &L
+	) -> Result<ChannelMonitor<<SP::Target as SignerProvider>::EcdsaSigner>, ChannelError>
+	where L::Target: Logger
+	{
+		if self.context.channel_state & !(ChannelState::MonitorUpdateInProgress as u32) != ChannelState::FundingCreated as u32 {
```

Comment:
Hmm, it looks like this is not updated to the ChannelState refactor, that is included in other parts already (of this same PR, same source file even).